rakelib/pg_connectivity_check.rake

```diff
@@ -0,0 +1,50 @@
+# frozen_string_literal: true
+
+namespace :db do
+  desc 'Check database connectivity and perform test operations'
+  task check: :environment do
+    puts '*** Test 1: Access Records'
+    begin
+      if UserAccount.exists?
+        puts 'UserAccount records exist.'
+      else
+        puts 'No UserAccount records found but database is up.'
+      end
+    rescue => e
+      Rails.logger.error(
```

Comment:
You can show the error as `puts "#e}"`. Rails.logger.error could potentially be misread by other devs

Reply:
oh 🤔 but I thought I wanted to send errors to datadog so we could set up a monitor for it.